### ISSUE TITLE
Creator Leaderboards

### ISSUE THREAD

**Issue by jvyden** [LittleBigRefresh/Refresh#44]
It would be cool to rank people by the amount of hearts or plays their levels have gotten.

It could encourage making levels, or you know, it could backfire and cause H4H to be an annoyance again.

**Comment by smugay** [comment]
What about people that just reupload levels rather than making their own original levels? Wouldn't it encourage creators to NOT create? Someone with no creating skills can just reupload a level from the old servers and get just as many hearts and plays as original levels.

**Comment by jvyden** [comment]
Yeah, good point. I was thinking of having a system to mark levels as reuploads to exclude statistics from those levels from the leaderboards - but I feel like that would be a fair bit of moderation work as it requires manual flagging either by a moderator or by a user. We could also probably flag against known root levels but I wouldn't know how to get a list of those (possibly from level backups taken from LittleBigArchive?)

It could also work for the category system - for example we could filter levels by if they're reuploads or not which I imagine people would appreciate.

**Comment by smugay** [comment]
The easiest way to cut down on moderation might be to mark profiles that exclusively reupload levels in some way, then you don't have to deal with manually moderating everything they upload

Yes, a category system would be really convenient! I recall the LBPU beacon moderators making a mistake where they team picked a level that was just a reupload, I don't imagine that mistake would have happened if there was a simple way to know if a level was re-uploaded

I absolutely love the idea of rewarding creators for making cool and original levels, and finding a way to have them co-exist with reuploads is just a challenge that comes along with such a system
